cpp/harvest/nacjd.cc

```diff
@@ -43,18 +43,31 @@ namespace {
 
 [[noreturn]] void Usage() {
     ::Usage(
-        "marc_title_in_file new_marc_title_out_file [count_of_rows]\n"
-        "where marc_title_in_file contains also icpsr records (001 or 035a)\n"
+        "[mode] [mode_params]\n"
+        "get_full input_file output_file\n"
+        "\t- input_file: contains also icpsr records (001 or 035a)\n"
         "these records are not processed any more\n"
-        "new_marc_title_out_file contains all icpsr records not contained in input file.\n"
-        "count_of_rows is an optional parameter. If it is absent, the number of requests will be equal to 9000)");
+        "\t- output_file contains all icpsr records not contained in input file.\n"
+        "\n"
+        "get_by_count input_file output_file number\n"
+        "\t- input_file: contains also icpsr records (001 or 035a)\n"
+        "these records are not processed any more\n"
+        "\t- output_file contains all icpsr records not contained in input file.\n"
+        "\t- number: is an optional parameter. If it is absent, the number of requests will be equal to 9000\n"
+        "\n"
+        "get_by_ID output_file ID\n"
+        "\t- output_file contains all icpsr records not contained in input file.\n"
+        "\t- ID: The NACJD of the dataset to download\n");
 }
 
 
 std::vector<std::string> ids_website;
 const unsigned int TIMEOUT_IN_SECONDS(15);
 const std::string NACJD_TITLES("/tmp/nacjd_titles.html");
+const std::string NACJD_TITLES_FILE_NAME("/nacjd_titles.html");
```

Comment:
Please remove all of these NACJD_... constants that contain and kind of file or path.